src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -2410,7 +2410,7 @@ struct MHAHelper {
         int32_t total_kv_len_aligned = 0;
         int32_t total_kv_len = 0;
         for (int32_t i = 0; i < seq_cout; i++) {
-            auto score_win_len = score_aggregation_window.ptr<int32_t>()[i];
+            auto score_win_len = score_aggregation_window ? score_aggregation_window.ptr<int32_t>()[i] : 0;
```

Comment:
According to the spec, when `score_aggregation_window` is unspecified, the value should be 1.